src/Form/Template.php

```diff
@@ -22,26 +21,74 @@
  */
 abstract class Template {
 	/**
+	 * Flag to check whether or not open success page in iframe.
+	 *
 	 * @var bool $openSuccessPageInIframe If set to false then success page will open in window instead of iframe.
 	 */
 	public $openSuccessPageInIframe = true;
 
 	/**
+	 * Flag to check whether or not open failed page in iframe.
+	 *
 	 * @var bool $openFailedPageInIframe If set to false then failed page will open in window instead of iframe.
 	 */
 	public $openFailedPageInIframe = true;
 
 	/**
-	 * @see src/Form/Template/LegacyFormSettingCompatibility.php:16 Check property description.
-	 * @var array $defaultSettings Form settings default values for form template.
+	 * Donation level display style
+	 *
+	 * @var string $donationLevelsDisplayStyle
+	 */
+	public $donationLevelsDisplayStyle = 'buttons';
+
+	/**
+	 * Donation form display style.
+	 *
+	 * @var string $donationFormDisplayStyle
 	 */
-	protected $defaultLegacySettingValues = [];
+	public $donationFormDisplayStyle = 'onpage';
 
 	/**
-	 * @see src/Form/Template/LegacyFormSettingCompatibility.php:18 Check property description.
-	 * @var array $mapToLegacySetting
+	 * Flag to check whether or not enable float label feature.
+	 *
+	 * @var string $enableFloatLabels
+	 */
+	public $enableFloatLabels = 'disabled';
+
+	/**
+	 * Continue to donation form button label.
+	 *
+	 * @var string $continueToDonationFormLabel
+	 */
+	public $continueToDonationFormLabel = '';
+
+	/**
+	 * Donation now button title.
+	 *
+	 * @var string $donateNowButtonLabel
+	 */
+	public $donateNowButtonLabel = '';
+
+	/**
+	 * Flag to check whether or not show donation form introduction text.
+	 *
+	 * @var string $showDonationIntroductionContent
+	 */
+	public $showDonationIntroductionContent = 'disabled';
+
+	/**
+	 * Donation introduction content position.
+	 *
+	 * @var string $donationIntroductionContentPosition
+	 */
+	public $donationIntroductionContentPosition = '';
```

Comment:
Please provide a description with what this does along with the possible values.